commands/build_lint.js

```diff
@@ -68,6 +68,16 @@ module.exports = function ({ output, lang, timeout }, sourceDir, { lint } = { li
       var pngIcons = results[3]
       var translations = results[4]
 
+      var zip = new yazl.ZipFile()
+
+      const safelyZipBuffer = (buf, name) => {
```

Comment:
Out of curiosity, when do we expect this to happen?

Reply:
I don't understand the question, I'm sorry xD.
If you're asking about errors (hence the `safelyZipBuffer`, `yazl` fails silently, hence the wrapping in a `try/catch`

Reply:
Sorry, I was unclear. I'm curious about (1) when an error would be thrown (2) why we want to ignore those errors.

Neither of these are blockers, I don't think.

Reply:
Sorry I didn't  answer this sooner. So, yazl will fail silently if there's an error in a file. I basically add that check so I could debug issues in files that we were reading. I don't know if we should crash if there's an error on a file, or just print a warning; but for know I was just printing a warning...

Reply:
I think we should throw an error instead of logging a warning because it makes it easier to diagnose problems. But I'm happy either way.